src/main/webapp/components/MultiTab.js

```diff
@@ -1,26 +1,34 @@
 import React, { Component } from 'react';
 import { hh, h } from 'react-hyperscript-helpers';
 import { Tabs, Tab } from 'react-bootstrap';
-
+import { isEmpty } from "../util/Utils";
 
 export const MultiTab = hh(class MultiTab extends Component {
 
   constructor(props) {
     super(props);
 
     this.state = {
-      key: ""
+      key: "",
+      defaultActive: ""
     };
 
     this.handleSelect = this.handleSelect.bind(this);
   }
 
- handleSelect(key) {
+  handleSelect(key) {
     this.setState({ key: key });
   }
 
   componentDidMount() {
-    this.setState({key: this.props.defaultActive});
+    this.setState({key: this.props.defaultActive, defaultActive: this.props.defaultActive});
+  }
+
+  static getDerivedStateFromProps(nextProps, prevState) {
+    if (nextProps.defaultActive && nextProps.defaultActive !== prevState.defaultActive){
+      return { key: nextProps.defaultActive, defaultActive: nextProps.defaultActive};
+    }
+    else return null;
```

Comment:
I think it's better with brackets
```suggestion
    else {
       return null;
     }
```